packages/ng/multi-select/input/select-input.component.html

```diff
@@ -2,9 +2,7 @@
 
 @if (!filterPillMode) {
 	@if (hasValue() && clearable && (disabled$ | async) === false) {
-		<button role="button" type="button" class="multipleSelect-clear clear" (click)="clearValue($event)">
-			<span class="pr-u-mask">{{ intl.clear }}</span>
-		</button>
+		<lu-clear class="multipleSelect-clear" (onClear)="clearValue($event)">{{ intl.clear }}</lu-clear>
```

Comment:
`onXXX` event names are not recommended to not be confused with Javascript events, might be better renaming it.